### PR TITLE
Adjust payment page responsive

### LINKED ISSUE #22801: Payment mode needs some adjusts on mobile view
Epic: https://github.com/PrestaShop/PrestaShop/issues/22358 

[Link to Figma prototype](https://www.figma.com/proto/b2sVogt6tyvB5EIkVmTAdP/PrestaShop-Responsive?node-id=608%3A0&scaling=scale-down)

#### Describe the bug

The page a bit broken on mobile 

image

#### Steps to Reproduce

1. Go to Payment Methods
2. See on mobile view (375px)

#### Additional information

* PrestaShop version: develop

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Payment page needs some ajust, an overflow on table and top card positionning
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #22801.
| How to test?      | Go on payment page on mobile, see issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22807)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/pages/_payment.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          padding-left: 5rem;
+
+          ul {
+            padding-left: 0;
+          }
+
+          .text-center {
+            // stylelint-disable-next-line
+            text-align: left !important;
+          }
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/pages/_payment.scss]: Really ?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/_payment.scss]: Yes really

There is a choice to do there :

- Do we use bootstrap responsive helper classes to text-align-left on mobile (So we separate the responsive to 2 context => Markup and CSS side
- Do we force it inside CSS side in order to have the responsive only on CSS side (more maintainable) ?

Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/_payment.scss]: I think it is always the same : Boostrap Helper Classes, else CSS !

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/_payment.scss]: It has been done there then : https://github.com/PrestaShopCorp/ps_checkout/pull/661

**Comment by NeOMakinG** [comment]
For the centered button, it's going to be left when this PR is accepted and checkout is released : https://github.com/PrestaShopCorp/ps_checkout/pull/661

**Comment by Progi1984** [comment]
> For the centered button, it's going to be left when this PR is accepted and checkout is released : [PrestaShopCorp/ps_checkout#661](https://github.com/PrestaShopCorp/ps_checkout/pull/661)

Checkout is not about the open source project.

**Review by Progi1984 — APPROVED** [review]

**Comment by NeOMakinG** [comment]
> > For the centered button, it's going to be left when this PR is accepted and checkout is released : [PrestaShopCorp/ps_checkout#661](https://github.com/PrestaShopCorp/ps_checkout/pull/661)
> 
> Checkout is not about the open source project.

And so?
You told me to stop caring about the paradigm of markup and CSS, so I need to modify some markup in order to modify some styles which is something I'm really not in favor of, then it forced me to work on another project in order to achieve the job :)

**Comment by Progi1984** [comment]
> > > For the centered button, it's going to be left when this PR is accepted and checkout is released : [PrestaShopCorp/ps_checkout#661](https://github.com/PrestaShopCorp/ps_checkout/pull/661)
> > 
> > 
> > Checkout is not about the open source project.
> 
> And so?
> You told me to stop caring about the paradigm of markup and CSS, so I need to modify some markup in order to modify some styles which is something I'm really not in favor of, it then forced me to work on another project in order to achieve the job :)

Do you fix all payment modules ?

**Comment by NeOMakinG** [comment]
> > > > For the centered button, it's going to be left when this PR is accepted and checkout is released : [PrestaShopCorp/ps_checkout#661](https://github.com/PrestaShopCorp/ps_checkout/pull/661)
> > > 
> > > 
> > > Checkout is not about the open source project.
> > 
> > 
> > And so?
> > You told me to stop caring about the paradigm of markup and CSS, so I need to modify some markup in order to modify some styles which is something I'm really not in favor of, it then forced me to work on another project in order to achieve the job :)
> 
> Do you fix all payment modules ?

At least the one installed by default on this page, and using a hook at the top

image

**Comment by Progi1984** [comment]
And for these ones ?
image

**Comment by NeOMakinG** [comment]
Same behavior than the whole BO:
image

Scroll in table

**Comment by Progi1984** [comment]
It's approved. Need a 2nd approved ;)

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM !!
**Before**
image
image
**After**
image
image
Thanks @NeOMakinG  !!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @SD1982

**Pull request merged**